layouts/shortcodes/card.html

```diff
@@ -5,6 +5,13 @@
 {{- $image := .Get "image" }}
 {{- $context := . -}}
 
+{{/*- Adding asset support for images here, so that Hugo can do its image processing magic. -*/}}
+{{- if not (urls.Parse $image).Scheme -}}
+  {{- with resources.Get $image -}}
```

Comment:
does it work with the case when users need to use images under `/static/` directory?